Neo4j.Driver/Neo4j.Driver/Internal/Connector/TcpSocketClient.cs

```diff
@@ -79,7 +79,7 @@ private Stream Stream
         {
             _uri = uri;
             _useTls = useTlsEncryption;
-            await _client.ConnectAsync(uri.Host, uri.Port).ConfigureAwait(true);
+            await _client.ConnectAsync(uri.Host, uri.Port).ConfigureAwait(false);
```

Comment:
No, this is the only place to have this value to be true. So I guess I will just change it to false then.